release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2997,4 +2997,37 @@ $ oc adm release info 4.13.4 --pullspecs
 [id="ocp-4-13-4-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-5"]
+=== RHSA-2023:3982 - {product-title} 4.13.5 bug fix and security update
+
+Issued: 2023-07-19
+
+{product-title} release 4.13.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:3982[RHSA-2023:3982] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3981[RHSA-2023:3981] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.5 --pullspecs
+----
+
+[id="ocp-4-13-5-bug-fixes"]
+==== Bug fixes
+* Previosly, the Gateway API feature did not provide DNS records with a trailing dot for the Gateway domain. This caused the status of the DNS record to never become available on the GCP platform. With this update, the DNS records for Gateway API Gateways are properly provisioned and the Gateway API feature works on GCP because the gateway service dns controller now adds a trailing dot if it is missing in the domain. (link:https://issues.redhat.com/browse/OCPBUGS-15434[*OCPBUGS-115434*])
```

Comment:
```suggestion
* Previously, the Gateway API feature did not provide DNS records with a trailing dot for the Gateway domain. This caused the status of the DNS record to never become available on the GCP platform. With this update, the DNS records for Gateway API Gateways are properly provisioned and the Gateway API feature works on GCP because the gateway service DNS controller now adds a trailing dot if it is missing in the domain. (link:https://issues.redhat.com/browse/OCPBUGS-15434[*OCPBUGS-115434*])
```
See the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_d